Userland/DevTools/Profiler/main.cpp

```diff
@@ -338,7 +338,7 @@ static bool prompt_to_stop_profiling(pid_t pid, DeprecatedString const& process_
 bool generate_profile(pid_t& pid)
 {
     if (!pid) {
-        auto process_chooser = GUI::ProcessChooser::construct("Profiler"sv, String::from_utf8_short_string("Profile"sv), Gfx::Bitmap::load_from_file("/res/icons/16x16/app-profiler.png"sv).release_value_but_fixme_should_propagate_errors());
+        auto process_chooser = GUI::ProcessChooser::construct("Profiler"sv, "Profile"sv), Gfx::Bitmap::load_from_file("/res/icons/16x16/app-profiler.png"_short_string.release_value_but_fixme_should_propagate_errors());
```

Comment:
Looks like some kind of accident happened here 😅

Reply:
oops, my regex skills were not prepared for this!